cmd/k8s-pipeliner/main.go

```diff
@@ -14,7 +14,7 @@ import (
 
 const (
 	// Version defines the current version of k8s-pipeliner
-	Version = "0.0.7"
+	Version = "0.0.8"
```

Comment:
this is annoying me. is it necessary? I can live without it from the output, or perhaps we can do something fancy at release-time?

Reply:
I'm down for "fancy at release time" but let's create an issue for that and proceed that way.